### PR TITLE
Make new completions specs pass

### PR DESCRIPTION
**Pull request by abe33**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -128,6 +128,6 @@
     "fbjs": "^0.8.6",
     "javascript-obfuscator": "^0.8.3",
     "sinon": "^2.3.5",
-    "editors-json-tests": "git://github.com/kiteco/editors-json-tests.git#master"
+    "editors-json-tests": "git://github.com/kiteco/editors-json-tests.git#new-completion-spec"
   }
```
Comment (jansorg) [inline package.json]: Revert this change because the json tests are in master now?

Comment (abe33) [reply package.json]: oh yeah right
